core/lib/zksync_core/src/state_keeper/batch_executor/mod.rs

```diff
@@ -37,8 +37,8 @@ pub(crate) enum TxExecutionResult {
     /// Successful execution of the tx and the block tip dry run.
     Success {
         tx_result: Box<VmExecutionResultAndLogs>,
-        tx_metrics: ExecutionMetricsForCriteria,
-        bootloader_dry_run_metrics: ExecutionMetricsForCriteria,
+        tx_metrics: Box<ExecutionMetricsForCriteria>,
```

Comment:
why do we need box here?

Reply:
So that enum variant sizes don't differ much. It's raised by clippy without `Box`

Reply:
I'd silence instead

Reply:
But why? Clippy error is completely legit and I consider wrapping into Box as an improvement